scipy/interpolate/_ndgriddata.py

```diff
@@ -30,9 +30,9 @@ class NearestNDInterpolator(NDInterpolatorBase):
 
     Parameters
     ----------
-    x : (Npoints, Ndims) ndarray of floats
+    x : (npoints, ndims) N-D ndarray of floats
```

Comment:
I'd read `N-D array` to mean it can be of any shape, while here it's clearly 2D.

Reply:
Changed to 2-D